<launch>

    <node name="igvc_visualizer" pkg="sb_pointcloud_processing" type="igvc_visualizer" output="screen">
	<rosparam param="h_margin_of_error"> 30 </rosparam>
	<rosparam param="s_margin_of_error"> 0.3 </rosparam>
	<rosparam param="v_margin_of_error"> 0.3 </rosparam>
	<rosparam param="image_update_rate"> 0.1 </rosparam>
        <remap from="/input_cloud" to="/camera/depth_registered/points" />
    </node>
</launch>
